drupal8-example/Makefile

```diff
@@ -0,0 +1,22 @@
+up:
+ifeq ($(shell uname), Darwin)
+	@echo "Looks like you're on macOS! Using docker-compose-mac.yml"
+	@docker-compose -f docker-compose.yml -f docker-compose-mac.yml up -d
+else
+	@docker-compose up -d
+endif
+
+	# Tada!! 🎉 Run 'make logs' to follow output.
+
+down:
+ifeq ($(shell uname), Darwin)
+	@echo "Looks like you're on macOS! Using docker-compose-mac.yml"
+	@docker-compose -f docker-compose.yml -f docker-compose-mac.yml stop
+	@docker-compose -f docker-compose.yml -f docker-compose-mac.yml down -v
```

Comment:
`--remove-orphans` is not necessary on Mac?

Reply:
I've added it.